mkdocs.yml

```diff
@@ -41,6 +41,7 @@ pages:
     - 'Classifier Calibration Plots': 'classifier_calibration.md'
     - 'Hyperparameter Tuning Effects': 'hyperpar_tuning_effects.md'
     - 'Out-of-Bag Predictions': 'out_of_bag_predictions.md'
+    - 'Spatial': 'spatial.md'
```

Comment:
If your headline ist "Handling of Spatial Data" it should have the same name here. Also please name the md file `handling_of_spatial_data.md`.

Reply:
ok :+1: